src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/DefaultComesLastCheck.java

```diff
@@ -71,15 +74,22 @@ public void visitToken(DetailAST ast) {
         final DetailAST defaultGroupAST = ast.getParent();
         //default keywords used in annotations too - not what we're
         //interested in
+        final Set<Integer> tokenTypes = new HashSet<>();
```

Comment:
Why do we need this `set` for an almost static list of 2 tokens?
Would it make more sense if we just moved this and condition on 89 to a new method? (no set, return boolean)